app/models/bgs_dependents/veteran.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module BGSDependents
+  class Veteran < Base
+    def initialize(proc_id, user)
+      @user = user
+      @proc_id = proc_id
+    end
+
+    def formatted_params(payload, user)
+      dependents_application = payload['dependents_application']
+      vet_info = [
+        *payload['veteran_information'],
+        ['first', user[:first_name]],
+        ['middle', user[:middle_name]],
+        ['last', user[:last_name]],
+        *dependents_application.dig('veteran_contact_information'),
+        *dependents_application.dig('veteran_contact_information', 'veteran_address'),
+        %w[vet_ind Y]
+      ]
+
+      if dependents_application['current_marriage_information']
+        vet_info << ['martl_status_type_cd', dependents_application.dig('current_marriage_information', 'type')]
+      end
+
+      vet_info.to_h
+    end
+
+    def veteran_response(participant, person, va_file_number, address, end_product)
```

Comment:
Not sure if you have access to the code climate report, it doesn't like that this method takes 5 params:
![image](https://user-images.githubusercontent.com/3077884/88835245-027f6800-d1a3-11ea-8e62-3e9e897a3326.png)